### DISCUSSION TITLE
我想申请下退款

### DISCUSSION THREAD

**Discussion by iam189cm** [getlantern/lantern-forum-cn#745] [General]
![IMG_5499](https://github.com/user-attachments/assets/d9d80fd5-757e-4163-a788-4aa71235f3a1)

我邮箱:83077071@qq.com
这是我的订单，用了几天感觉不太好用。

**Comment by MaxMKMax** [comment]
@iam189cm 
好的. 我帮您提交问题.

**Reply by iam189cm** [reply]
谢谢
